docs/other_integrations.rst

```diff
@@ -59,3 +59,10 @@ Requests
 --------
 
 .. automodule:: ddtrace.contrib.requests
+
+.. _grpc:
+
+Grpc
+^^^^
```

Comment:
Having this as `^^^^` will nest it under `Requests` in the TOC/menu, which we do not want.

```suggestion
----
```